sinks/dashboard/dashboard.py

```diff
@@ -539,27 +551,37 @@ def closeEvent(self, event):
             button_layout.addWidget(save_changes)
             button_layout.addWidget(discard_changes)
             button_layout.addWidget(cancel)
-
+            # Apply layout to save popup
             save_layout.addLayout(button_layout)
-
             save_popup.setLayout(save_layout)
-            # Handle user click events.
+
+
+            # Connect buttons to action listeners. 
```

Comment:
Should I consider adding constants for these values here? And if so would these be global constants or is there a certain file where constants go.

Reply:
I think these should be just specific to this file, and I would use an enum for this